searchlib/src/vespa/searchlib/queryeval/filter_wrapper.h

```diff
@@ -55,6 +55,9 @@ class FilterWrapper : public SearchIterator {
     BitVector::UP get_hits(uint32_t begin_id) override {
         return _wrapped_search->get_hits(begin_id);
     }
+    BitVectorIterator * asBitVector() noexcept override {
+        return _wrapped_search->asBitVector();
```

Comment:
using the inner iterator will invalidate the outer one. I do not think this is a good idea.

Reply:
Very good point, fixed PTAL